R/uvhydrograph-render.R

```diff
@@ -363,3 +352,180 @@ YEndpoint <- function (corr.value.sequence, uncorr.value.sequence) {
   
   return(y.endpoint)
 }
+
+
+#' TODO
+#' @importFrom grDevices rgb
+getPrimaryPlotConfig <- function(reportObject, info, correctionLabels, dataSides, dataLimits) {
+  styles <- getUvStyles()
+  
+  x <- reportObject[[1]]$time
+  y <- reportObject[[1]]$value
+  
+  comp_lbl <- info$comp_UV_lbl
+  comp_type <- info$comp_UV_type
+  legend.name <- reportObject[[1]]$legend.name
+  plotStartDate <- info$plotStartDate
+  plotEndDate <- info$plotEndDate
+  
+  corrArrowPositions <- getCorrectionArrowPositions(x, y, correctionLabels)
+  corrAblinePositions <- getCorrectionAbLines(reportObject)
+  
+  primary_lbl <- info$primary_lbl
+  reference_lbl <- info$reference_lbl
+  comp_lbl <- paste("Comparison", info$comp_UV_TS_lbl, "@", info$comp_UV_lbl)
+  
+  compAxes <- TRUE
+  compAnnotations <- TRUE
+  compLabel <- primary_lbl
```

Comment:
what is the difference between comp_lbl and compLabel? Can we have some comments specifying?